examples/nidmm_measurement/measurement.py

```diff
@@ -31,27 +32,7 @@
 )
 service_options = ServiceOptions()
 
-
-class Function(Enum):
-    """Wrapper enum that contains a zero value."""
-
-    NONE = 0
-    DC_VOLTS = nidmm.Function.DC_VOLTS.value
-    AC_VOLTS = nidmm.Function.AC_VOLTS.value
-    DC_CURRENT = nidmm.Function.DC_CURRENT.value
-    AC_CURRENT = nidmm.Function.AC_CURRENT.value
-    TWO_WIRE_RES = nidmm.Function.TWO_WIRE_RES.value
-    FOUR_WIRE_RES = nidmm.Function.FOUR_WIRE_RES.value
-    FREQ = nidmm.Function.FREQ.value
-    PERIOD = nidmm.Function.PERIOD.value
-    TEMPERATURE = nidmm.Function.TEMPERATURE.value
-    AC_VOLTS_DC_COUPLED = nidmm.Function.AC_VOLTS_DC_COUPLED.value
-    DIODE = nidmm.Function.DIODE.value
-    WAVEFORM_VOLTAGE = nidmm.Function.WAVEFORM_VOLTAGE.value
-    WAVEFORM_CURRENT = nidmm.Function.WAVEFORM_CURRENT.value
-    CAPACITANCE = nidmm.Function.CAPACITANCE.value
-    INDUCTANCE = nidmm.Function.INDUCTANCE.value
-
+FunctionWrapper = generate_wrapper_enum(nidmm.Function)
```

Comment:
FYI, this is a tradeoff: dynamically generated enums don't have type annotations according to https://stackoverflow.com/a/74335189 so mypy can't check whether `FunctionWrapper.SDKFJDKFJ` is valid and VS Code can't auto-complete `FunctionWrapper.` into `FunctionWrapper.DC_VOLTS`.

I'm not sure whether this is more or less error-prone than listing out all of the enum values like in your previous PR. Neither seems ideal.

It would be more convenient if users could specify an enum that doesn't a 0 value and nims would automatically translate 0 -> default_value.

Reply:
And yeah, mypy can't handle enums with dynamic members.
https://mypy-play.net/?mypy=latest&python=3.11&gist=63649673382c32dfd9d77b912040997a
![image](https://github.com/ni/measurementlink-python/assets/17914399/153cd137-10b7-4cc7-a6c1-5b237f8df102)

Reply:
Hold on, I have another idea I want to try out

Reply:
https://docs.python.org/3/howto/enum.html#restricted-enum-subclassing explains why the obvious solution is not supported:
```
class FunctionWrapper(nidmm.Function):
    NONE = 0
```

It would be nice if we could do this:
```
class FunctionWrapper(nidmm.Function):
    @classmethod
    def _missing_(cls, value: object) -> Any:
        if value == 0:
            return cls.RED
        return super()._missing_(value)
```

However, subclassing an enum that has values is not supported, even if the subclass doesn't define any values.

Reply:
```
class FunctionWrapper(nidmm.Function):
    NONE = 0
```

Yeah, I was slightly annoyed that python doesn't support enum subclassing like this since it feels so natural.